dcos-commons/src/test/java/org/apache/mesos/scheduler/DefaultSchedulerTest.java

```diff
@@ -245,7 +245,7 @@ public String getName() {
         register();
 
         Plan plan = defaultScheduler.getPlan();
-        Assert.assertTrue(inExpectedState(plan, Arrays.asList(Status.PENDING, Status.COMPLETE, Status.PENDING)));
+        Assert.assertTrue(inExpectedState(plan, Arrays.asList(Status.PENDING, Status.PENDING, Status.PENDING)));
```

Comment:
Pretty sure adding the `CustomExecutor` should not have changed any test behavior here.  This looks like a bug was introduced.